sherpa/utils/__init__.py

```diff
@@ -1752,7 +1752,7 @@ def quantile(sorted_array, f):
 
     Based on the description of the GSL routine
     gsl_stats_quantile_from_sorted_data - e.g.
-    http://www.gnu.org/software/gsl/manual/html_node/Median-and-Percentiles.html
+    https://www.gnu.org/software/gsl/manual/html_node/Median-and-Percentiles.html
```

Comment:
perhaps the link is now- https://www.gnu.org/software/gsl/doc/html/statistics.html#median-and-percentiles